### DISCUSSION TITLE
FB overvoltage and FB overcurrent

### DISCUSSION THREAD

**Discussion by agadave** [tbnobody/OpenDTU#1714] [General]
Hello,
What do "FB overvoltage" and "FB overcurrent" errors mean? What is "FB"
Thank

**Comment by MetaChuh** [comment]
afawk, fb is related to the flyback transformer within the inverter.
both messages given (overvoltage and overcurrent), are for the dc input(s) of the inverter.

they should never appear, unless you have following dc wiring issues:
1) the solar cells you have connected, can either exceed the inverter's voltage or amperage current.
2) some of the inverter's dc inputs are bridged, with either y-cables or any other possibility to bridge the positive lead of one dc input with the positive lead of another dc input (same for the negative ports)
3) there is a battery connected to one or more of the inputs, that can deliver more volt or ampere, than the maximum allowed for this inverter.
4) any combination of the above

**Reply by agadave** [reply]
My inverter dealer has just informed me: the voltage on my network is too high. I have 240V-243V. I'm near the transformer. The dealer tells me that with a hoymiles dtu I can configure the value. Is this possible with opendtu? Thank

**Comment by dumpfheimer** [comment]
I had this today, a few minutes after startup. At the time the panels where generating ~1W each (of the 6) so I do not think it could have been overcurrent by the panels. I have a single panel per string, no battery, no Y cables. I also get frequent grid fluctuation entries (very often 2 min past the hour) - but seemingly only or mostly in summertime. 

Will keep an eye on this

**Comment by gibzwein** [comment]
I have "FB overcurrent". Battery connected to input
